### ISSUE TITLE
Illegal week of the year

### ISSUE THREAD

**Issue by dkahia** [paolotremadio/homebridge-automation-calendar#17]
I’ve been receiving a warning that the week of the year exceeds the maximum - week 53 of 52. It doesn’t seem to be creating any issues, but thought I’d go ahead and report it.

**Comment by paolotremadio** [comment]
Yes, that’s just a warning. HomeKit still works as expected. It is due to one of the fields being declared with max value of 52 when sometimes the year can go up to 53 weeks. I’ll get around to fix it eventually but it doesn’t affect functionality.
